stow/sass/style-child-theme-editor.scss

```diff
@@ -32,6 +32,12 @@
  */
 @import "../../varia/sass/blocks/editor";
 
+/**
+ * Full Site Editing
+ * - Full Site Editing overrides
+ */
+ @import "full-site-editing-editor";
+
```

Comment:
I've generally added the FSE overrides at the very bottom of the child theme files, after the extras, in the off-chance that those extras affect the FSE styles making it a pain to debug.